rockspec/apisix-master-0.rockspec

```diff
@@ -34,7 +34,7 @@ dependencies = {
     "lua-resty-ctxdump = 0.1-0",
     "lua-resty-dns-client = 5.2.0",
     "lua-resty-template = 1.9",
-    "lua-resty-etcd = 1.4.3",
+    "lua-resty-etcd = 1.4.4",
```

Comment:
AFAIK, the health check is not enabled by default.

Reply:
I can add health checks to apisix